qiskit_aer/backends/controller_wrappers.py

```diff
@@ -0,0 +1,29 @@
+import importlib
+
+
+def try_import_backend(backend_module_suffix):
+    module_name = f".controller_wrappers_{backend_module_suffix}"
+    try:
+        return importlib.import_module(module_name, "qiskit_aer.backends")
+    except ImportError:
+        return None
+
+
+IMPORTED_BACKEND = None
+BACKENDS = ["cuda", "rocm", "cpu"]
+
+for backend_suffix in BACKENDS:
```

Comment:
One could also add a environment variable to for example force the CPU implementation.

But that can be done in a followup PR.